End2End/functest/src/main/functions/queueTriggerTest/function.json

```diff
@@ -0,0 +1,13 @@
+{
+    "scriptFile": "../../../../target/azure-functions-java-sample-1.0-SNAPSHOT.jar",
+    "entryPoint": "com.microsoft.serverless.QueueTests.queueTriggerTest",
+    "bindings": [
```

Comment:
Same here..what is the difference between queueTrigger and queueTriggerTest

Reply:
Sorry for the dupe like queueTrigger and queueTriggerTest. The folders contain "*Test" should have 
been deleted, but seems they were left behind with my careless mistake.